docs/components-README.md

```diff
@@ -82,3 +82,33 @@ Simple documentation for components being used and a sample of their `animations
   ...
 }
 ```
+
+## Visual Pane
+
+- `animations_json` - object
+  - `blackboardData`- object
+    - `componentType` - Needs to be the string "visual-pane"
```

Comment:
Can we list the possible values for componentType somewhere in this readme?

Reply:
This is a good idea; once we have all the components we need, I will add an enum validation to the backend for ensuring it is only one of those when objects are modified.